src/Sets/1f416f67-b167-4b61-a8cc-55f2e99251a7/set.xml

```diff
@@ -0,0 +1,2185 @@
+			<property name="Attack Zone" value="" />
+			<property name="Damage" value="" />
+			<property name="Hand Size" value="" />
+			<property name="Vitality" value="" />
+			<property name="Keywords" value="" />
+			<property name="Split Difficulty" value="" />
+			<property name="Split Rules" value="" />
+			<property name="Split Keywords" value="" />
+			<property name="Format" value="Legacy - Eternal - Standard - Turbo" />
+		</card>
+		<card id="55ab78b0-8d29-4913-a162-62b98730714a" name="Requiem">
+			<property name="Difficulty" value="3" />
+			<property name="Rarity" value="UR" />
+			<property name="Control" value="6" />
+			<property name="Type" value="Asset" />
+			<property name="CardText" value="E [Once per turn]: This attack gets +1 damage for each of your opponent's face down foundations.  E Commit: This attack gets -1 damage for each of your opponent's face down foundations. Your opponent may flip 1 foundation to cancel this effect." />
+			<property name="Resources" value="Earth Good Void" />
+			<property name="Block Modifier" value="1" />
+			<property name="Block Zone" value="Mid" />
+			<property name="Speed" value="" />
+			<property name="Attack Zone" value="" />
+			<property name="Damage" value="" />
+			<property name="Hand Size" value="" />
+			<property name="Vitality" value="" />
+			<property name="Keywords" value="Unique Weapon" />
+			<property name="Split Difficulty" value="" />
+			<property name="Split Rules" value="" />
+			<property name="Split Keywords" value="" />
+			<property name="Format" value="Legacy - Eternal - Standard - Turbo" />
+		</card>
+		<card id="efb67363-24b5-4811-9233-ff98e121e7e3" name="Deep Scar">
+			<property name="Difficulty" value="0" />
+			<property name="Rarity" value="UR" />
+			<property name="Control" value="2" />
+			<property name="Type" value="Attack" />
+			<property name="CardText" value="E Discard 1 momemtum: Unflip up to 3 of your opponent's face down foundations. This attack gets +X damage. X equals the total printed control values of the unflipped cards. Your opponent may flip 3 foundations to cancel this effect." />
+			<property name="Resources" value="Earth Good Void" />
+			<property name="Block Modifier" value="0" />
+			<property name="Block Zone" value="Mid" />
+			<property name="Speed" value="6" />
+			<property name="Attack Zone" value="Mid" />
+			<property name="Damage" value="6" />
+			<property name="Hand Size" value="" />
+			<property name="Vitality" value="" />
+			<property name="Keywords" value="Breaker:3 Fury Weapon" />
+			<property name="Split Difficulty" value="" />
+			<property name="Split Rules" value="" />
+			<property name="Split Keywords" value="" />
+			<property name="Format" value="Legacy - Eternal - Standard - Turbo" />
+		</card>
+		<card id="20efb394-165e-4d55-8bc4-6211c0c0995c" name="Reborn Grinder">
+			<property name="Difficulty" value="4" />
+			<property name="Rarity" value="C" />
+			<property name="Control" value="3" />
+	
```

Comment:
Try using the &amp; reference in place of 'and'.  I forgot that directly adding '&' to the name portion would mess with the compiling.  I'm trying to keep the '&' in so the deck editor plug-in will work better with UFS Ultra and they tend to be 1:1 in terms of what's printed on the card vs what they put in the database.